docs/readthedocs/models/evcs_model.md

```diff
@@ -0,0 +1,29 @@
+(evcs_model)=
+## EvCs Model
```

Comment:
What is your opinion about not using an abbreviation, here?

Reply:
I completely agree.